src/foam/nanos/google/api/sheets/import/GoogleSheetsImportConfig.js

```diff
@@ -94,6 +94,11 @@ foam.CLASS({
       class: 'foam.mlang.ExprProperty',
       hidden: true,
       javaJSONParser: 'foam.lib.json.ExprParser.instance()',
+    },
+    {
+      name: 'isUnitProperty',
+      class: 'Boolean',
```

Comment:
Add documentation: to explain what a unitProperty is.

Reply:
added